assets/tests/components/lateView.test.tsx

```diff
@@ -135,7 +130,7 @@ describe("LateView", () => {
     expect(mockDispatch).toHaveBeenCalledWith(closeLateView())
   })
 
-  test("clicking ghost run number opens ghost and sends Fullstory event", () => {
+  test("clicking ghost run number opens ghost and sends Fullstory event", async () => {
```

Comment:
Small nitpick, but if you're updating this line anyway it should probably refer to Google Tag Manager, not Fullstory.